py/picca/tests/delta_extraction/data/.config.ini

```diff
@@ -49,11 +50,12 @@ limit eta = (0.5, 1.5)
 limit var lss = (0.0, 0.3)
 num bins variance = 20
 num iterations = 5
-num processors = 1
 order = 1
 use constant weight = False
 use ivar as weight = False
 out dir = /Users/iperezra/software/picca/py/picca/tests/delta_extraction/results/config_tests/
+num processors = 1
```

Comment:
This should not be neeeded, right? This value should be on the general section only. Not sure why you remove it in line 52 and re-add it here

Reply:
it is, I tried out the tests (without that num processors which I originally deleted and then re-added at possible a different position) and as we're copying the value into the structure it's written for general and all children (ie data and expected flux for this case).

Reply:
if you mean that changing the position should not be needed: that is true...